Document crash_loop_limit node configuration for 23.1 (#1131)
redpanda-data/documentation: modules/manage/pages/iceberg/redpanda-topics-iceberg-snowflake-catalog.adoc

To configure your Redpanda cluster to enable Iceberg on a topic and integrate with Open Catalog:

ifdef::env-cloud[]
. xref:manage:iceberg/use-iceberg-catalogs.adoc#store-a-secret-for-rest-catalog-authentication[Store the Open Catalog client secret in your cluster] using `rpk` or the Data Plane API.

> . Edit your cluster configuration to set the `iceberg_enabled` property to `true`, and set the catalog integration properties listed in the example below using `rpk` or the Control Plane API. For example, to use `rpk cluster config set`, run the following:

Reviewer: ```suggestion
. Edit your cluster configuration to set the `iceberg_enabled` property to `true`, and set the catalog integration properties listed in the example below using `rpk` or the Control Plane API. For example, to use `rpk cluster config set`, run:
```

+
Use your own values for the following placeholders:
+
- `<snowflake-orgname>` and `<open-catalog-account-name>`: Your https://docs.snowflake.com/en/sql-reference/sql/create-catalog-integration-open-catalog#required-parameters[Open Catalog account URI^] is composed of these values.
+
TIP: In Snowflake, navigate to **Admin**, then **Accounts**. Click the ellipsis near your Open Catalog account name, and select **Manage URLs**. The **Current URL** contains `<snowflake-orgname>` and `<open-catalog-account-name>`.
- `<open-catalog-connection-client-id>`: The client ID of the service connection you created in an earlier step.
- `<open-catalog-connection-client-secret>`: The client secret of the service connection you created in an earlier step.
- `<open-catalog-name>`: The name of your catalog in Open Catalog.
+
[,bash,role=no-copy]
----
Successfully updated configuration. New configuration version is 2.
----

. You must restart your cluster so that the configuration changes take effect.
endif::[]

> . Enable the integration for a topic by configuring the topic property `redpanda.iceberg.mode`. This mode creates an Iceberg table for the topic consisting of two columns, one for the record metadata including the key, and another binary column for the record's value. See xref:manage:iceberg/about-iceberg-topics.adoc#enable-iceberg-integration[Enable Iceberg integration] for more details on Iceberg modes.

Reviewer: ```suggestion
. Enable the integration for a topic by configuring the topic property `redpanda.iceberg.mode`. This mode creates an Iceberg table for the topic consisting of two columns: one for the record metadata including the key, and another binary column for the record's value. See xref:manage:iceberg/about-iceberg-topics.adoc#enable-iceberg-integration[Enable Iceberg integration] for more details on Iceberg modes.
```

ifdef::env-cloud[]
+
Use any of the following to set `redpanda.iceberg.mode`:
+

> * `rpk`. See the examples below for running the `rpk topic` commands.

Reviewer: ```suggestion
* `rpk`. See the following examples to run `rpk topic` commands.
```

* The Cloud UI. Navigate to *Topics* to create a new topic and specify `redpanda.iceberg.mode` in *Additional Configuration*, or edit an existing topic under the topic's *Configuration* tab.
* The Data Plane API to xref:api:ROOT:cloud-dataplane-api.adoc#post-/v1/topics[create a new topic] or xref:api:ROOT:cloud-dataplane-api.adoc#patch-/v1/topics/-topic_name-/configurations[update a property for an existing topic]. Specify the key-value pair for `redpanda.iceberg.mode` in the request body.
endif::[]
+

> The following examples show how to use xref:get-started:rpk-install.adoc[`rpk`] to either create a new topic, or alter the configuration for an existing topic, to set the Iceberg mode to `key_value`.  

Reviewer: ```suggestion
The following examples show how to use `rpk` to create a new topic or alter the configuration for an existing topic, setting the Iceberg mode to `key_value`.  
```
